packages/xo-server/src/xapi-stats.js

```diff
@@ -244,7 +229,38 @@ const STATS = {
 }
 
 // -------------------------------------------------------------------
-
+// RRD
+// json: {
+//   meta: {
+//     start: Number,
+//     step: Number,
+//     end: Number,
+//     rows: Number,
+//     columns: Number,
+//     legend: String []
+//   },
+//   data: [{
+//      t: Number,
+//      values: Number[],
+//    }]
+// }
+
+// Local cache
+// _statsByObject : {
+//   [uuid]: {
+//     [step]: {
+//       endTimestamp: json.meta.end,
+//       localTimestamp: getCurrentTimestamp(),
```

Comment:
IMHO it would be better to say what the meaning of this value and what we use it for instead of where we get it from.